### PR TITLE
Updated path handling in examples/preprocessing/otp.py

### LINKED ISSUE #10835: update path handling in examples/preprocessing/otp.py
all of our `data_path` functions now return `pathlib.Path` objects, which allows things like the following:

https://github.com/mne-tools/mne-python/blob/37a9e5587443109515a85cdc4396af4e014ecf82/examples/datasets/brainstorm_data.py#L32-L35

Compare this to the old pattern (using `os.path.join`, often imported as `op.join`):

```py
raw_path = os.path.join(data_path, 'MEG', 'bst_raw',
                        'subj001_somatosensory_20111109_01_AUX-f.ds')
```

or the *really* old pattern (using the `+` operator on plain strings):

```py
raw_path = (data_path + '/MEG/bst_raw/' +
            'subj001_somatosensory_20111109_01_AUX-f.ds')
```

Update the paths in the file mentioned in the issue title to use the new pathlib-style, which is considered the modern, best-practice way to handle file paths.

### PR DESCRIPTION
**Pull request by architsinghal-mriirs**
Thanks for contributing a pull request! Please make sure you have read the
[contribution guidelines](https://mne.tools/dev/install/contributing.html)
before submitting.

Please be aware that we are a loose team of volunteers so patience is
necessary. Assistance handling other issues is very welcome. We value
all user contributions, no matter how minor they are. If we are slow to
review, either the pull request needs some benchmarking, tinkering,
convincing, etc. or more likely the reviewers are simply busy. In either
case, we ask for your understanding during the review process.

Again, thanks for contributing!

#### Reference issue
Fixes #10835

#### What does this implement/fix?
Updated the path handling in examples/preprocessing/otp.py to the modern standard.

#### Additional information
Any additional information you think is important.

### CODE REVIEW

File: examples/preprocessing/otp.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 dipole_number = 1
 data_path = bst_phantom_elekta.data_path()
 raw = read_raw_fif(
-    op.join(data_path, 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
+    (data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif'))
```
Comment (drammock) [inline examples/preprocessing/otp.py:38]:
```suggestion
    data_path / 'kojak_all_200nAm_pp_no_chpi_no_ms_raw.fif')
```
remove extra parentheses

Comment (architsinghal-mriirs) [reply examples/preprocessing/otp.py:38]: The parenthesis is required for the 'read_raw_fif' import right?

Comment (drammock) [reply examples/preprocessing/otp.py:38]: you only need one set of parentheses here. What I've suggested only removes the inner set.

**Comment by drammock** [comment]
superseded by #10894

**Pull request closed without merging**